src/Todo.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import User from './User';
+
+function Todo({ todo: { user, title, completed } }) {
+  return (
+    <>
+      <User user={user} />
```

Comment:
```suggestion
      <User userData={user} />
```
user user user читается не очень хорошо